streamly.cabal

```diff
@@ -664,58 +664,58 @@ test-suite Prelude.MaxRate
   else
     buildable: False
 
-test-suite array-test
+test-suite Data.Array.Storable.Foreign
   import: test-options
   type: exitcode-stdio-1.0
   main-is: Streamly/Test/Array.hs
```

Comment:
This test module name also needs to in sync with the source module name.

Reply:
We can have a common module and include that in each array module, defining the module specific cpp macro before we include it.